Gemfile

```diff
@@ -9,6 +9,8 @@ gem 'bootstrap_form', '~> 5.1'
 
 gem 'bullet'
 
+gem 'cancancan'
```

Comment:
[OPTIONAL] Kindly add MIT license to your project.

Reply:
resolved